frontend/src/components/AddConcepts.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
## Occasionally getting harmless console errors - couldn't replicate
When I first loaded the app in local, I got a bunch of console errors, but I didn't screenshot.

Then, I refreshed the frontend and went back to the add-concepts page, and got `DataGetter.jsx:107 Uncaught (in promise) Error: AxiosError: Request failed with status code 422`

<details><summary>Screenshot</summary>
<p>

<img width="1506" alt="Screenshot 2024-04-26 at 6 34 01 PM" src="https://github.com/jhu-bids/TermHub/assets/13045020/9568a9f9-e2fc-4953-bbba-fa6560ef6db9">

</p>
</details> 

In neither case was did my experience as a user result in any noticeable issue, though.

After I restarted the backend and frontend again, I got no more console errors, interestingly enough.